rancher-k8s-images/k8s/entry.sh

```diff
@@ -39,6 +39,9 @@ CONTAINERIP=$(curl -s http://rancher-metadata/2015-12-19/self/container/ips/0)
 if [ "$1" == "kubelet" ]; then
     /usr/bin/share-mnt /var/lib/kubelet /sys -- kubelet-start.sh "$@"
 elif [ "$1" == "kube-apiserver" ]; then
+    curl -sSL https://github.com/rancher/hostname-update-service/releases/download/v0.1.0/hostname-update-service > /hostname-update-service
```

Comment:
Let's not download on start, but in the Dockerfile build instead.  Might as well put the migration in a standard location too like `/usr/local/bin`.   Also we need to find an approach that if this service dies it is restarted.